docs/resources/agent.md

```diff
@@ -46,13 +46,15 @@ resource "kubernetes_pod" "dev" {
 
 ### Optional
 
+- `allow_login_before_ready` (Boolean) Allow users to login to the workspace before the agent is ready. Note that, when enabled, the agent could still be executing the startup script and the workspace in an incomplete state when logging in.
```

Comment:
@kylecarbs does this match what you were suggesting in https://github.com/coder/coder/issues/5749#issuecomment-1385877505?

I interpreted it as an option to allow/prevent it. Is that how you intended it? Or did you intend it to be the "default behavior for login before ready"? If it's the latter, we could rename this to be just `login_before_ready` instead of allow.

Reply:
I might invert the naming scheme because, eventually we'll make this default to blocking login before ready, and ideally it'd be false by default.

Apart from that we're aligned!